### PR TITLE
Assets 98994

### PR DESCRIPTION
**Pull request by mdickson-adbe**
Make the 'deliverables' tab of program overview dynamic. Also made some adjustments to the 'overview' tab based on feedback.

- Before: https://main--assets-distribution-portal--adobe.hlx.page/sample-public-site
- After: https://assets-98994--adobe-gmo--hlxsites.hlx.page/drafts/mdickson/campaigns

### CODE REVIEW

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 1]
```diff
@@ -781,37 +394,48 @@ function buildHeaderRow(category, headerType, isInactive) {
     return headerRow;
 }
 
-function buildTableRow(campaignJson, createHidden) {
-    //console.log(campaignJson);
+function buildTableRow(deliverableJson, kpi, createHidden) {
     const dataRow = document.createElement('div');
     dataRow.classList.add('row', 'datarow');
     if (createHidden) dataRow.classList.add('inactive');
+    const status = (deliverableJson.deliverableStatusUpdate == null) ? "??" : deliverableJson.deliverableStatusUpdate + "%";
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:401]: instead of ?? can you add Not Available?

Comment (mdickson-adbe) [reply blocks/gmo-campaign-details/gmo-campaign-details.js:401]: Done. :)

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 2]
```diff
@@ -781,37 +394,48 @@ function buildHeaderRow(category, headerType, isInactive) {
...
-        <div class='property table-column column2'>${campaignJson.type}</div>
-        <div class='property table-column column3'>${campaignJson.channel}</div>
+        <div class='property table-column column1 deliverable-name'>${deliverableJson.deliverableName}</div>
+        <div class='property table-column column2'>${deliverableJson.deliverableType}</div>
+        <div class='property table-column column3 platforms'>${platformString}</div>
         <div class='property table-column column4'>
-            <a href="${campaignJson.reviewLinkHref}" class="campaign-link">Review Link</a>
+            <a href="${deliverableJson.reviewLink}" class="campaign-link" target="_blank">Review Link</a>
         </div>
         <div class='property table-column column5'>
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:414]: @mdickson-adbe looks Final Asset anchor tag is removed. we have `linkedFolderLink` in the API response to show the link

Comment (mdickson-adbe) [reply blocks/gmo-campaign-details/gmo-campaign-details.js:414]: There's some logic farther down that appends the anchor tag if `linkedFolderLink` isn't null- see line 433

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 3]
```diff
@@ -781,37 +394,48 @@ function buildHeaderRow(category, headerType, isInactive) {
...
+                    <div class='status-bar' style='width: ${status}'></div>
                 </div>
             </div>
         </div>
-        <div class='property table-column column8'>${campaignJson.dueDate}</div>
-        <div class='property table-column column9'>${campaignJson.driver}</div>
+        <div class='property table-column column8'>${checkBlankString(deliverableJson.taskCompletionDate)}</div>
+        <div class='property table-column column9'>${checkBlankString(deliverableJson.driver)}</div>
     `
+    if (!(deliverableJson.linkedFolderLink == null)) {
```
[the remark this answers is not in the document]
Comment (mdickson-adbe) [reply blocks/gmo-campaign-details/gmo-campaign-details.js:432]: Fixed, the anchor tag wasn't being given text content before.

**Review by shiv-gup — APPROVED** [review]

**Pull request merged**